engine/schema/src/com/cloud/upgrade/dao/Upgrade41100to41110.java

```diff
@@ -66,6 +66,68 @@ public boolean supportsRollingUpgrade() {
     @Override
     public void performDataMigration(Connection conn) {
         updateSystemVmTemplates(conn);
+        markUnnecessarySecureConfigsAsUnsecure(conn);
+    }
+
+    private void markUnnecessarySecureConfigsAsUnsecure(Connection conn) {
```

Comment:
What if you explain in a documentation here the history behind this method, and why it has been implemented?
I mean, also, why only these parameters.

Reply:
sure, do you ask for javadoc or comments?

Reply:
Yes. Something small, only to explain why this code is important. Otherwise, it might be hard in the future to understand why we are doing this.

Reply:
does this satisfy your reuirements, @rafaelweingartner

Reply:
Well, I did not see anything yet in the Github. However, if you add just a snippet of comments explaining why we need such method, I am fine with it.